### PR TITLE
removing outdated alert

### PR DESCRIPTION
**Pull request by asteurer**
It looks like the `KUBECONFIG` env var is automatically exporting, so this alert is no longer necessary.

### CODE REVIEW

File: content/en/docs/Getting started/_index.md
Comment (jimmykarily) [inline content/en/docs/Getting started/_index.md:117]: @mauromorales removing it here, doesn't remove it from the old versions of the docs, right? It would be good for people to still see if if they are visiting old versions of the docs.

**Review by jimmykarily — APPROVED** [review]

**Comment by jimmykarily** [comment]
Nice @asteurer , thanks for the PR!

**Comment by asteurer** [comment]
@jimmykarily Looks like I don't have write access. Will you merge this when you have the chance?

**Pull request merged**
